mdb-radio-list.html

```diff
@@ -85,11 +85,14 @@
           type: String,
           reflectToAttribute: true,
           value: 'large',
-          reflectToAttribute: true,
           observer: '_enumSize'
         },
       },
 
+            changeHandler: function (e) {
+                this.fire('recommendation-changed-Top', {"vale": e.target.innerText});
```

Comment:
We should consider using a generic event name (maybe something like "radio-button-tapped") because mdb-radio-list is a generic component which does not need to know what recommendation is.